### PR TITLE
Hello_Page: Microverse_First_Project_Week_One

### PR DESCRIPTION
**Pull request by Dinma-Faith-zz**
-Created HTML and CSS file
-Added Github folder, workflows folder with linters.yml file inside the folders
-Added .gitignore file
-Added test.md in .gitignore file
-Added a README file

### CODE REVIEW

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @Dinma-Faith , 

Impressive work so far! 
There are some issues that you still need to work on to go to the next project but you are almost there!

### 📝👌 Your project highlights 🔥😎

- Splendid work with the README file 👀

## Required Changes ♻️

- [ ] Please make sure that linters are checked in the pull request and all linters pass, linters result should be green

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,27 @@
...
+
+
+## Built With
+
+- HTML
+- CSS
+
+
+
+
```
Comment (elmar8287) [inline README.md:11]: - [ ] [OPTIONAL] To make your repo more informative, you can add instructions on how to start with your project on the local machine, using the `git clone` command.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Hello Page</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+   <!--header-->
+<body>
+  <header>
+    <h1 class="header">"Hello Microverse!"</h1>
```
Comment (elmar8287) [inline index.html:13]: - [ ] [OPTIONAL] You can use the text "Hello Microverse" without parentheses 👍

File: index.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Hello Page</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+   <!--header-->
```
Comment (elmar8287) [inline index.html:10]: - [ ] [OPTIONAL] You can remove comments from your code to make it more clear 👍

**Review by elmar8287 — APPROVED** [review]
Hi @Dinma-Faith, 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**